source/GameData.cpp

```diff
@@ -106,6 +106,10 @@ namespace {
 	map<const Sprite *, double> solarWind;
 	Set<News> news;
 	map<string, vector<string>> ratings;
+	vector<string> outfitCategories;
+	vector<string> shipCategories = {
+	"Fighter",
```

Comment:
I'd either single-line this, or at minimum indent, i.e.
`vector<string> shipCategories = { "Fighter", "Drone" };`
or
```
vector<string> shipCategories = {
    "Fighter", "Drone"
};
```